### PR TITLE
🆕Add client to add, remove, and fetch a dict of emoji

### PR DESCRIPTION
**Pull request by nifengle**
I hate the emoji requirement

To use: 

```python
from emoji import client

client.emoji_dict()
```

### CODE REVIEW

File: emoji_client.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import os
+import sys
+import slack
+
+
+client = slack.WebClient(token=os.environ['SLACK_API_TOKEN'])
```
Comment (gmkohler) [inline emoji_client.py]: so if I 
```py
import emoji_client
```
then this remains private and `emoji_client.add_emoji`, etc. are exposed?  Or is this exposed as a global in the importing file?

File: emoji_client.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+def add_emoji(filepath, name):
+    return client.api_call(
+        "emoji.add",
+        files={"image": file},
+        data={"name": name, "mode":"data"}
+    )
+
+
+def remove_emoji(name):
+    return client.api_call("emoji.remove", data={"name":"thoughts-and-prayers", "mode":"data"})
```
Comment (gmkohler) [inline emoji_client.py]: we should use the function argument here instead of `"thoughts-and-prayers"`

File: emoji_client.py [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import sys
+import slack
+
+
+client = slack.WebClient(token=os.environ['SLACK_API_TOKEN'])
+
+def add_emoji(filepath, name):
+    return client.api_call(
+        "emoji.add",
+        files={"image": file},
```
Comment (gmkohler) [inline emoji_client.py]: should this be `filepath`?

File: emoji_client.py (hunk 1 above)
Comment (nifengle) [reply emoji_client.py]: this would be exposed. I've changed the structure up a bit so that an instance of the client is exposed

**Pull request merged**